neuralpredictors/layers/readouts.py

```diff
@@ -988,11 +1003,24 @@ def forward(self, x, sample=None, shift=None, out_idx=None):
         if shift is not None:
             grid = grid + shift[:, None, None, :]
 
-        y = F.grid_sample(x, grid, align_corners=self.align_corners)
-        y = (y.squeeze(-1) * feat).sum(1).view(N, outdims)
+        if not multiplex:
+            y = F.grid_sample(x, grid, align_corners=self.align_corners)
+            y = (y.squeeze(-1) * feat).sum(1).view(N, outdims)
+        else:
+            y = torch.einsum("ncwh,uco->nwho", x, feat)
```

Comment:
Maybe it is just me, but I can never immediately say what the einsum exactly is supposed to do without extensively thinking about it. You could add a small comment explaining what the einsum is supposed to do.